jobs/travelnest_edinburgh_20180604.html

```diff
@@ -0,0 +1,54 @@
+---
+title: Software Development Engineers (All levels)
+company: TravelNest
+url: https://travelnest.breezy.hr/p/8f72c74d67bd01-software-development-engineers-all-levels
+location: Edinburgh,  United Kingdom
+contract: permanent
+contact:
+    name: Kim Watson
+    email: kim.watson@travelnest.com
+    phone: +447398971079
+    an engineer's name: Laurence Billingham
+    an engineer's email: laurence.billingham@travelnest.com
```

Comment:
The main contact details are for our head of talent but we'd also like to let applicants 
talk to someone already in the role (me).
Please let me know if there is a better way to format these detail

Reply:
I suggest that you put this content in the body of the advert. Leave a single contact in the header, it will look much better